README.md

```diff
@@ -101,8 +101,14 @@ The API supports passing a custom handling in that case, to mimic the behaviour
 truncate the password.
 
 ```java
-BCrypt.with(LongPasswordStrategies.truncate()).hash(6, new byte[100]);
-BCrypt.with(LongPasswordStrategies.hashSha512()).hash(6, new byte[100]); //allows to honour all pw bytes
+BCrypt.with(LongPasswordStrategies.truncate()).hash(6, pw);
+BCrypt.with(LongPasswordStrategies.hashSha512()).hash(6, pw); //allows to honour all pw bytes
+```
+
+Don't forget to use the same strategy when verifying:
+
+```java
+BCrypt.verifyer(LongPasswordStrategies.truncate()).verify(pw, hash)
```

Comment:
You are missing a semicolon here 😉 
Talking about it, I think the interface of the lib a bit surprising. Why `BCrypt.with()...` is for hashing, and `BCrypt.verifyer()...` is for verifying? 
It would really make more sense, from an external point of view, if we could simply do:
`BCrypt.with(LongPasswordStrategies.truncate()).verify(pw, hash);`

I get that, _internally_, it is 2 different classes (too keep the code tidy I guess?) but it's not really obvious when using the lib.

Reply:
I see your point. My understanding was that hashing and verifying are usually done in different places, so normally one would only do the one or other. I don't like to change the API lightly so I suggest we keep this fix with the current API and I'll create an improvement task where we can think about changing the API in a major version update.

Reply:
Of course, let's keep focus on this PR.

Don't know if you seen my first sentence _"You are missing a semicolon here"_
It was kind of buried in my rambling 😅

Reply:
Yes, just commit the update ;)